src/v0.1.0/properties/structures/_cheminfo_inchikey.yaml

```diff
@@ -12,8 +12,8 @@ type:
   - "string"
   - "null"
 description: |-
-  The standard InChIKey identifier of the structure, as laid out by the InChI Trust (https://www.inchi-trust.org).
-  Standard InChIKey is not guaranteed to be unique and in extremely rare cases the same InChIKey may be assigned to several different structures.
+  An InChIKey identifier of the structure, as laid out by the InChI Trust (https://www.inchi-trust.org).
+  It can be any InChIKey: not necessary a Standard InChIKey, but can also be the latter.
 examples:
   - "QZAYGJVTTNCVMB-UHFFFAOYSA-N"
```

Comment:
It might be nice to change this to a non-standard InChI.

Reply:
Good idea, added in 667565f399bd4723664dbc2a9c260ca7ceb8dc35.